Formula/haproxy.rb

```diff
@@ -32,6 +32,40 @@ def install
     bin.install "haproxy"
   end
 
+  def caveats; <<~EOS
+    To start haproxy with brew services, first create and edit your config at:
+      #{etc}/haproxy.cfg
+    Logs can be found at:
+      #{var}/log/haproxy.log
+    EOS
+  end
+
+  plist_options :manual => "haproxy"
```

Comment:
should this also have the -f?

Reply:
Yea, let me rather just remove this.

Reply:
Ah, I see, that's a good idea, let me add this.